### PR TITLE
feat(ui): add settings sub program

### PR DESCRIPTION
**Pull request by gigovich**

### CODE REVIEW

File: client/cmd/root.go [hunk 1]
```diff
@@ -56,6 +57,7 @@ func init() {
 	}
 	rootCmd.PersistentFlags().StringVar(&daemonAddr, "daemon-addr", defaultDaemonAddr, "Daemon service address to serve CLI requests [unix|tcp]://[path|host:port]")
 	rootCmd.PersistentFlags().StringVar(&managementURL, "management-url", "", fmt.Sprintf("Management Service URL [http|https]://[host]:[port] (default \"%s\")", internal.ManagementURLDefault().String()))
+	rootCmd.PersistentFlags().StringVar(&adminURL, "admin-url", "https://app.wiretrustee.com", "Admin Panel URL [http|https]://[host]:[port]")
```
Comment (mlsmaycon) [inline client/cmd/root.go]: Default should be https://app.netbird.io

File: client/ui/client_ui.go [hunk 2]
```diff
@@ -226,6 +385,40 @@ func (s *serviceClient) client() (proto.DaemonServiceClient, error) {
 	return s.conn, nil
 }
 
+// getSrvConfig from the service to show it in the settings window.
+func (s *serviceClient) getSrvConfig() {
+	s.managementURL = "https://api.netbird.io:33073"
+	s.adminURL = "https://app.wiretrustee.com"
```
Comment (mlsmaycon) [inline client/ui/client_ui.go]: should be:
```GO
s.managementURL = "https://api.wiretrustee.com:33073"
s.adminURL = "https://app.netbird.io"
```

**Review by mlsmaycon — CHANGES_REQUESTED** [review]
some changes for the default Admin panel URL during rebranding

File: client/cmd/root.go (hunk 1 above)
Comment (gigovich) [reply client/cmd/root.go]: Fixed

File: client/ui/client_ui.go (hunk 2 above)
Comment (gigovich) [reply client/ui/client_ui.go]: Fixed

**Review by mlsmaycon — APPROVED** [review]

**Pull request merged**